Source/JavaScriptCore/runtime/JSGenericTypedArrayViewPrototypeFunctions.h

```diff
@@ -250,12 +250,24 @@ ALWAYS_INLINE EncodedJSValue genericTypedArrayViewProtoFuncIncludes(VM& vm, JSGl
         }
     }
 
-    if constexpr (ViewClass::elementSize == 1 && ViewClass::Adaptor::isInteger) {
-        if (index < length) {
-            auto* result = static_cast<typename ViewClass::ElementType*>(memchr(array + index, targetOption.value(), length - index));
-            return JSValue::encode(jsBoolean(!!result));
+    if constexpr (ViewClass::Adaptor::isInteger) {
+        if constexpr (ViewClass::elementSize == 1) {
+            if (index < length) {
+                auto* result = static_cast<typename ViewClass::ElementType*>(memchr(array + index, targetOption.value(), length - index));
+                return JSValue::encode(jsBoolean(!!result));
+            }
+            return JSValue::encode(jsBoolean(false));
         }
-        return JSValue::encode(jsBoolean(false));
+
+#if CPU(ARM64)
```

Comment:
I kind of wish this was more like `HAVE(memchr16)` instead of `CPU(ARM64)`.

Reply:
Changed.